mpmp/utilities/classify_utilities.py

```diff
@@ -115,7 +115,7 @@ def run_cv_stratified(data_model,
                 max_iter=cfg.max_iter
             )
         except ValueError as e:
-            if 'Only one class' in str(e):
+            if ('Only one class' in str(e)) or ('got 1 class' in str(e)):
```

Comment:
Is this catching errors thrown by sklearn and rewrapping them so you can filter them later?

Reply:
Yeah, exactly. I don't want this error to completely stop the script that calls this function (which it does if it isn't caught); I just want to skip the gene that this happens for, and move on to the next gene after logging the error.

I'm not totally sure why there are two different types of single-class errors that get thrown in this sklearn function, but I'm fine with treating them the same here.